src/constants/event-emitter.ts

```diff
@@ -22,6 +22,9 @@
  * SOFTWARE.                                                                      *
  **********************************************************************************/
 
-export const PROBE_RESPONSE_RECEIVED = 'PROBE_RESPONSE_RECEIVED'
-export const PROBE_RESPONSE_VALIDATED = 'PROBE_RESPONSE_VALIDATED'
-export const PROBE_STATUS_PROCESSED = 'PROBE_STATUS_PROCESSED'
+export const RESPONSES_READY_TO_PROCESS = 'RESPONSES_READY_TO_PROCESS' // 1. probe done, process responses
```

Comment:
I think the previous name `PROBE_RESPONSE_RECEIVED` is better as an event name

Reply:
noted. thank you.